From: Alexis Sukrieh <sukria@sukria.net>
Date: Thu, 26 Nov 2015 14:32:16 -0800
Forwarded: not-needed
Subject: debian-version

backup-manager's version string includes "-debian" We want people to know that
it differs from upstream.
---
 backup-manager | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backup-manager b/backup-manager
index 15ba0f4..2712606 100755
--- a/backup-manager
+++ b/backup-manager
@@ -27,7 +27,7 @@ set -e
 
 RELEASE="true"
 REVISION='$Revision$'
-VERSION="0.7.11"
+VERSION="0.7.11-debian1"
 
 #Set prefix for paths
 BIN_PREFIX=/usr/bin
